### PR TITLE
zeromq: update to 4.3.1

### PR DESCRIPTION
**Pull request by ndowens**

### CODE REVIEW

**Comment by maxice8** [comment]
@Vaelatern

File: srcpkgs/zeromq/template [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
-checksum=8e9c3af6dc5a8540b356697081303be392ade3f014615028b3c896d0148397fd
+checksum=bcbabe1e2c7d0eec4ed612e10b94b112dd5f06fcefa994a0c79a45d835cd21eb
 
 zeromq-devel_package() {
 	depends="${sourcepkg}-${version}_${revision}"
 	short_desc+=" - development files"
 	pkg_install() {
 		vmove usr/include
 		vmove usr/lib/pkgconfig
-		vmove usr/share/man/man3
```
Comment (Vaelatern) [inline srcpkgs/zeromq/template]: Why? I don't see a changelog item in zmq for this

Comment (ndowens) [reply srcpkgs/zeromq/template]: If you are talking about the man page, I don’t understand why man page is in devel package

Comment (ndowens) [reply srcpkgs/zeromq/template]: Ah didn’t know that, makes sense now

**Comment by ndowens** [comment]
Done

File: srcpkgs/zeromq/template (hunk 1 above)
Comment (Vaelatern) [reply srcpkgs/zeromq/template]: thank you.

**Comment by ndowens** [comment]
@Vaelatern sorry for the trouble, just seen a manpage in devel pkg and thought it was a mistake 🙂

**Comment by Vaelatern** [comment]
We all learn what section 3 is eventually.

**Pull request merged**